cdap-data-fabric/src/test/java/io/cdap/cdap/spi/data/sql/SqlStructuredTableRegistryTest.java

```diff
@@ -41,7 +41,8 @@
   public static void beforeClass() throws Exception {
     pg = PostgresInstantiator.createAndStart(TEMP_FOLDER.newFolder());
     DataSource dataSource = pg.getPostgresDatabase();
-    registry = new SqlStructuredTableRegistry(dataSource);
+    // Setting fetch size to 100 for testing
+    registry = new SqlStructuredTableRegistry(dataSource, 100);
```

Comment:
Can we use a value that's more like what we plan to use in real life? e.g. 10 or even 1

Reply:
Changed to 10.